.gitlab/llnl-ci.yml

```diff
@@ -129,7 +130,7 @@ sdk-test:
       done
     - spack load exaworks py-pytest
     - export RADICAL_PILOT_DBURL=${MONGODB_CONNECTION_STRING}?tlsAllowInvalidCertificates=true
```

Comment:
just a comment regarding keeping `RADICAL_PILOT_DBURL` within GitLab's Settings - CI/CD - Variables: it doesn't let to keep string value with `?` character and having it masked, that's why for such value we have this extra exporting